packages/eslint-config-change-base/package.json

```diff
@@ -15,20 +15,20 @@
   },
   "devDependencies": {
     "@change-org/eslint-plugin-change": "^1.1.0",
-    "eslint": "^5.16.0",
-    "eslint-config-prettier": "^6.11.0",
-    "eslint-plugin-import": "^2.22.0",
-    "eslint-plugin-jest": "^23.18.0",
+    "eslint": "^7.18.0",
+    "eslint-config-prettier": "^7.2.0",
+    "eslint-plugin-import": "^2.22.1",
+    "eslint-plugin-jest": "^24.1.3",
     "eslint-plugin-lodash": "^7.1.0",
-    "eslint-plugin-mocha": "^7.0.1",
-    "eslint-plugin-prettier": "^3.1.4",
+    "eslint-plugin-mocha": "^8.0.0",
+    "eslint-plugin-prettier": "^3.3.1",
     "eslint-plugin-promise": "^4.2.1",
     "eslint-plugin-security": "^1.4.0",
-    "prettier": "^2.0.5"
+    "prettier": "^2.2.1"
   },
   "peerDependencies": {
     "@change-org/eslint-plugin-change": "1.x",
-    "eslint": "5.x >=5.15.2",
+    "eslint": ">=5.15.2",
     "eslint-plugin-import": "2.x >=2.16.0",
```

Comment:
Should we relax these rules as well?  For packages like this I rather allow all major versions and have it break, than assume that it's not compatible and have to update the package each time a new major version is released.  But I'm not sure how breaking changes might surface with a package like this.

Reply:
I'm ok with that, yeah... It would make this package easier to use.  With a possible exception for peerDependencies with known vulnerabilities, though.  If there are known vulnerable versions of a peerDep then I'd still prefer to "encourage" people to upgrade those.

All versions of eslint-plugin-import look safe from vulns, for example:
https://snyk.io/vuln/npm:eslint-plugin-import

By this logic, we could make the `eslint` constraint `>=5' since there are [known vulns in 4.x](https://snyk.io/vuln/npm:eslint).

Reply:
Cool.  I simplified the allowed versions to be >= major version only.  I checked the other packages and didn't see any vulnerabilities on them.  I'm not aware of any hard dep on the specific versions that were listed, but it's safe to say that we're on or above those versions now.